packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -142,7 +152,7 @@ export const zoomMeetingsSchema = z.object({
 });
 
 const zoomTokenSchema = z.object({
-  scope: z.literal("meeting:write"),
+  scope: z.string().regex(new RegExp("meeting:write")),
```

Comment:
This should validate scope has meeting:write access, whether its meeting:write or meeting:write:admin